mkosi/__init__.py

```diff
@@ -6923,6 +6923,7 @@ def setup_network_veth(args: MkosiArgs, root: Path, do_run_build_script: bool, c
                     [Match]
                     Virtualization=!container
                     Type=ether
+                    Driver=virtio_net
```

Comment:
If this is really for veth, then this should be "Driver=veth", no?
When I create veth, then networkd detects its driver as "veth".
```
systemd-networkd[1335053]: veth99: Saved new link: ifindex=240, iftype=ETHER(1), kind=veth
systemd-networkd[1335053]: veth99: Saved permanent hardware address: 00:00:00:00:00:00
systemd-networkd[1335053]: veth99: Found driver: veth
systemd-networkd[1335053]: veth99: Saved original MTU 1500 (min: 68, max: 65535)
systemd-networkd[1335053]: veth99: Saved hardware address: 8e:e0:63:d0:aa:70
systemd-networkd[1335053]: veth99: Saved IPv6 link-local address generation mode: eui64
systemd-networkd[1335053]: veth99: Flags change: +MULTICAST +BROADCAST
```
If this is correct, then maybe adjust the function name 'setup_network_veth'. Or please add a comment about this.